pulpcore/cli/common/generic.py

```diff
@@ -724,7 +718,7 @@ def _type_callback(ctx: click.Context, param: click.Parameter, value: Optional[s
         "Search for {entities} with these content hrefs in them (JSON list or "
         "@file containing a JSON list)"
     ),
-    callback=load_json_callback,
+    callback=json_callback,
```

Comment:
What do you think about the following naming convention?
`load_string_callback` searches for "@" and attempts to load from file else value.
`json_callback` transforms `value` to json.
`load_json_callback` is `json_callback` decorated (chained) with `load_string_callback`.

Reply:
`string_callback` would be the same as specifying nothing...